main.tex

```diff
@@ -140,14 +140,28 @@ \subsection{Active hyperlinks}
 a page break (often the error message does not identify this as the underlying problem). In such cases, then simplest solution is to find the
 offending link and enclose it within \verb|\mbox{}|. Note that the {\tt hypperref} package has a `draft' option that will disable all linking; this can be useful to get a draft paper to compile when La\TeX\ has split such a link. This is a better solution while a paper is in development, since the offending link may no longer fall on a page break once all material has been added to the paper.
 
-\section{How to Refer to DESC and LSST} 
+\section{How to Refer to DESC, LSST, the Rubin Observatory, the Telescope, and Camera} 
+
+This section is informed by the name usage guide for the Vera C. Rubin Observatory\footnote{\url{https://project.lsst.org/documents/name-use-guide}} and guidelines developed among the LSST science collaborations.
```

Comment:
```suggestion
This section is informed by the name usage guide for Vera C. Rubin Observatory\footnote{\url{https://project.lsst.org/documents/name-use-guide}} and guidelines developed among the LSST science collaborations\footnote{\url{http://bit.ly/RubinLSSTSCsnaming}}.
```

Reply:
Thank you, Rachel.  Do we expect that the guidelines developed among the LSST science collaborations will be staying in that Google doc or promoted to something with a more permanent looking URL?

Reply:
That's meant to be the permanent home.